content/blog/urbit-for-normies.md

```diff
@@ -0,0 +1,181 @@
++++
+title = "Urbit for Normies"
+date = 2020-02-12
+description = "What is Urbit and why should you care?"
+[extra]
+author = "Erik Newton"
+ship = "~patnes-rigtyn"
+image ="https://media.urbit.org/site/posts/essays/normies-2.png"
++++
+
+## What is Urbit and why should you care?
+
+*A layperson’s guide to the coming new internet.*
```

Comment:
Looking at it in preview, these two lines here should go — they're description content, and they hang weirdly at the top.